pkg/bgp/server.go

```diff
@@ -32,15 +32,19 @@ func NewBGPServer(c *Config) (b *Server, err error) {
 
 	if err = b.s.StartBgp(context.Background(), &api.StartBgpRequest{
 		Global: &api.Global{
-			As:         c.AS,
+			Asn:        c.AS,
 			RouterId:   c.RouterID,
 			ListenPort: -1,
 		},
 	}); err != nil {
 		return
 	}
 
-	if err = b.s.MonitorPeer(context.Background(), &api.MonitorPeerRequest{}, func(p *api.Peer) { log.Println(p) }); err != nil {
+	if err = b.s.WatchEvent(context.Background(), &api.WatchEventRequest{Peer: &api.WatchEventRequest_Peer{}}, func(r *api.WatchEventResponse) {
```

Comment:
This was taken from https://github.com/osrg/gobgp/commit/5798cfaf265c1358b75be2d574f1652dee9f199b